tests/sentry/notifications/test_notificationcontroller.py

```diff
@@ -55,6 +56,7 @@ def add_notification_setting_provider(
 # The tests below are intended to check behavior with the new
 # NotificationSettingOption and NotificationSettingProvider tables,
 # which will be enabled with the "organization:notification-settings-v2" flag.
+@control_silo_test(stable=True)
```

Comment:
@scefali i think i saw in your pr that you were planning to add this later, should i take this out for now?

Reply:
@isabellaenriquez no, i wasn't touching that file. Not a bad idea to add in your PR